pkg/uninstall/uninstall.go

```diff
@@ -97,3 +111,37 @@ func (obj *uninstaller) createWatcher(ctx context.Context, pkg ctrlpkg.Package)
 		return nil, fmt.Errorf("unexpected object kind: %v", pkg.GroupVersionKind().Kind)
 	}
 }
+
+func DeleteNamespace(ctx context.Context, namespace string) {
+	clientset := cliutils.KubernetesClient(ctx)
+	err := clientset.CoreV1().Namespaces().Delete(ctx, namespace, metav1.DeleteOptions{})
+	if err != nil {
+		fmt.Fprintf(os.Stderr, "❌ error deleting namespace: %v\n", err)
+		cliutils.ExitWithError()
+	}
+
+	fmt.Printf("\nNamespace %s has been deleted.\n", namespace)
```

Comment:
please use uninstaller.status for printing status message. also we usually print status message before making an operation. E.g. "Deleting namespace %s" instead of "Namespace %s deleted"